community/graphdb-api/src/main/java/org/neo4j/graphdb/spatial/impl/CRS.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphdb.spatial.impl;
```

Comment:
This is a new pattern to `impl` packages - if this is implementation, why is it part of the API?

Reply:
I don't care about the package name, but I want to provide implementations to users of Core API and procedure writers. And these implementations should not be from the Values module.